api_tests/src/post.spec.ts

```diff
@@ -135,7 +135,8 @@ test('Update a post', async () => {
 test('Sticky a post', async () => {
   let postRes = await createPost(alpha, betaCommunity.community.id);
 
-  let stickiedPostRes = await stickyPost(alpha, true, postRes.post_view.post);
+  let betaPost1 = (await resolvePost(beta, postRes.post_view.post)).post;
+  let stickiedPostRes = await stickyPost(beta, true, betaPost1.post);
```

Comment:
This was wrong, alpha is not a mod and shouldnt be able to make a post sticky. It only worked due to a bug in the apub fetcher which is fixed now.

Reply:
Alpha is an admin, and I just checked the server code, admins should be able to sticky.

Reply:
Admin on instance alpha, but not on beta. So the sticky wont federate.